src/Storages/StorageSQLite.cpp

```diff
@@ -144,7 +144,9 @@ class SQLiteSink : public SinkToStorage
 
         sqlbuf << ") VALUES ";
 
-        auto writer = FormatFactory::instance().getOutputFormat("Values", sqlbuf, metadata_snapshot->getSampleBlock(), storage.getContext());
+        auto write_context = Context::createCopy(storage.getContext());
+        write_context->setSetting("output_format_values_escape_quote_with_quote", Field(true));
```

Comment:
may be we can copy, modify and save the context in advance (in constructor) not to do that on each consume(). even though it will not make any noticeable improvement, just more clean.

Reply:
Indeed, make total sense, thanks. Rebased.